cmd/fuseml_core-cli/main.go

```diff
@@ -33,6 +33,9 @@ func main() {
 	)
 	{
 		addr = *addrF
+		if addr == "" {
+			addr, _ = os.LookupEnv("CORE_SERVER_URL")
```

Comment:
just one tiny request: make it `FUSEML_SERVER_URL`, or `FUSEML_SERVER` please. This will help with the transition to the work I'm currently doing to introduce cobra and viper to the CLI. Viper has great support for 12-factor apps, including env variables and config files.